roles/openshift_prometheus/tasks/install_prometheus.yaml

```diff
@@ -0,0 +1,223 @@
+---
+
+# namespace
+- name: Add prometheus project
+  oc_project:
+    state: "{{ state }}"
+    name: "{{ openshift_prometheus_namespace }}"
+    node_selector: "{{ openshift_prometheus_node_selector | replace(':','=') | replace('{', '') | replace('}', '') | replace(' ', '') | replace(\"u\'\", '') | replace(\"\'\", '') }}"
```

Comment:
@joelddiaz cool!
I was unaware of that option.
much cleaner.
I'll check it out